### PR TITLE
fix: Path error while setting up shifu demos in local system

### PR DESCRIPTION
**Pull request by SuperAayush**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Adding an instruction in order to hit the right directory to make shifu tutorials work without throwing incorrect path errors.

#### Which issue(s) this PR fixes:

Fixes https://github.com/Edgenesis/shifu/issues/597

### CODE REVIEW

**Comment by SuperAayush** [comment]
> #### What type of PR is this?
> /kind documentation
> 
> #### What this PR does / why we need it:
> Adding an instruction in order to hit the right directory to make shifu tutorials work without throwing incorrect path errors.
> 
> #### Which issue(s) this PR fixes:
> Fixes [Edgenesis/shifu#597](https://github.com/Edgenesis/shifu/issues/597)

I have mentioned the directory name as shifudemos instead of testdir, it will automatically be fixed once the second part of this issue is solved: https://github.com/Edgenesis/shifu/issues/597

**Comment by tomqin93** [comment]
Hi @SuperAayush , thank you for this PR, in order to merge this in we need to adjust https://github.com/Edgenesis/shifu/blob/main/test/scripts/shifu-demo-install.sh#L34 too, do you mind creating a PR in https://github.com/Edgenesis/shifu for this?

**Comment by SuperAayush** [comment]
> Hi @SuperAayush , thank you for this PR, in order to merge this in we need to adjust https://github.com/Edgenesis/shifu/blob/main/test/scripts/shifu-demo-install.sh#L34 too, do you mind creating a PR in https://github.com/Edgenesis/shifu for this?

Hi @tomqin93, I have asked the same question by commenting on the issue here:
https://github.com/Edgenesis/shifu/issues/597#issuecomment-1553013455

The PR is ready can you look at that comment and let me know how to proceed?

File: docs/tutorials/demo-try.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ When installing ***Shifu*** installer, five virtual devices are created and conn
 
 ## Preparation
 
+To try out the devices mentioned below make sure to install shifu in your local system from [Quick Install](https://shifu.dev/docs/tutorials/demo-install) before proceeding forward.
```
Comment (tomqin93) [inline docs/tutorials/demo-try.md:22]: shifu => ***Shifu***

**Review by tomqin93 — COMMENTED** [review]
@SuperAayush added 1 minor comment, the rest looks good

**Comment by SuperAayush** [comment]
> @SuperAayush added 1 minor comment, the rest looks good

Thank you for the review, resolved the issue!!

**Comment by SuperAayush** [comment]
> Hi @SuperAayush , thank you for this PR, in order to merge this in we need to adjust https://github.com/Edgenesis/shifu/blob/main/test/scripts/shifu-demo-install.sh#L34 too, do you mind creating a PR in https://github.com/Edgenesis/shifu for this?

For this I have made a PR, please review it once you find time: https://github.com/Edgenesis/shifu/pull/604

Thank you!!

**Review by tomqin93 — APPROVED** [review]
LGTM

**Comment by tomqin93** [comment]
Thanks @SuperAayush , will merge this in after we release v0.19.0 on Monday

**Review by tomqin93 — APPROVED** [review]

**Pull request merged**